README.md

```diff
@@ -0,0 +1,345 @@
+---
+permalink: /guides/guide-codewind/
+layout: guide-markdown
+title: Getting Started with Codewind and Kabanero
+duration: 30 minutes 
+description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use.
+tags: ['kabanero', 'appsody', 'codewind', 'microservice']
+projectid: guide-codewind
+page-layout: guide-markdown
+page-duration: 30 minutes 
+page-releasedate: 02-21-2020
+page-guide-category: stacks
+page-essential: true
+page-essential-order: 1
+page-description: Take advantage of Codewind's tools to help build high quality cloud native applications regardless of which IDE or language you use. 
+guide-author: Kabanero
+page-tags: ['kabanero', 'appsody', 'codewind', 'microservice']
+page-permalink: /guides/{projectid}
+repo-description: TBD
+---
+
+# Getting Started with Codewind and Kabanero
+
+## Objectives
+
+### Learning objectives 
+
+In this guide you will learn how to develop a simple microservice with your chosen IDE, using Eclipse Codewind, an open source project, which provides IDE extensions for popular IDEs (VS Code and Eclipse IDE).  Eclipse Codewind provides the ability to create new projects based on application stack hub that include application stacks customized to meet your company policies and consistently deploy applications and microservices at scale.
```

Comment:
I would avoid mentioning stack hub as we do not define what one of these is anywhere. How about 

"Eclipse Codewind provides the ability to create new projects based on application stacks that can be customized to meet your company policies and consistently deploy applications and microservices at scale."